_Boards/Atmel/Board_Mega/MFBoards.h

```diff
@@ -1,39 +1,54 @@
+//
+// MFBoards.h (Arduino Mega)
+//
+// (C) MobiFlight Project 2022
+//
+
 #ifndef MFBoardMega_h
 #define MFBoardMega_h
 
 #ifndef MF_SEGMENT_SUPPORT
-#define MF_SEGMENT_SUPPORT 1
+#define MF_SEGMENT_SUPPORT   1
```

Comment:
Please don't reformat because it will be listed as change and makes review harder. Thanks.